packages/pci-sdk-iframe/src/types/IActionType.d.ts

```diff
@@ -0,0 +1,3 @@
+type IActionType = 'HIDE_DATA' | 'EMIT_VISIBILITY_MESSAGE' | 'SET_ERROR' | 'SET_CARD_DATA' | 'SET_LOADING' | 'SET_THEME';
```

Comment:
Is there any way (and value in) trying to use `IActionType` to set the type property in the `IAction.d.ts` interfaces?

Reply:
I've tried but no idea about how to do it :S